src/Selenium2Library/keywords/element.py

```diff
@@ -407,10 +439,12 @@ def drag_and_drop(self, source, target):
         Examples:
         | Drag And Drop | elem1 | elem2 | # Move elem1 over elem2. |
         """
-        src_elem = self._element_find(source, True, True)
-        trg_elem = self._element_find(target, True, True)
-        ActionChains(self._current_browser()).drag_and_drop(src_elem, trg_elem).perform()
+        src_elem = self.element_find(source, True, True)
+        trg_elem = self.element_find(target, True, True)
```

Comment:
General comments not related to the architecture change:

- `element_find()` clearly needs default values so that `True, True` doesn't need to be repeated.
- In general Boolean arguments should be given in named format like `something=True` to make it easier to understand the calling code.
- This module is rather big. Should consider splitting it into smaller pieces. Element finding logic could perhaps be moved into context.

Because the above aren't really related to the architecture change, it's probably better not to do it as part of this PR. Could perhaps just add some TODOs.